mir-module/src/main/resources/META-INF/resources/js/mir/accesskey.js

```diff
@@ -0,0 +1,244 @@
+const RSC_URL = webApplicationBaseURL + "rsc/accesskey/";
+let labels = undefined;
+
+function getParameterByName(name, url = window.location.href) {
+    name = name.replace(/[\[\]]/g, '\\$&');
+    var regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+    results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, ' '));
+}
+
+function generateKey(plength) {
+	var keylistalpha = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+	var keylistint = "123456789";
+	var keylistspec = "!@#_%$";
+	var temp = '';
+	var len = plength / 2;
+	var len = len - 1;
+	var lenspec = plength - len - len;
+
+	for (i = 0; i < len; i++)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'for' condition.](https://app.codacy.com/gh/MyCoRe-Org/mir/pullRequest?prid=7162253)
```suggestion
	for (i = 0; i < len; i++)
```